HW04/aleksei_zhuchkov/spec/notification_spec.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require_relative '../lib/notification'
+
+RSpec.describe Notification do
+  let(:notification) { described_class.new('test title', 'test description') }
+
+  it 'return title' do
+    expect(notification.title).to eq('test title')
+  end
+
+  it 'return description' do
+    expect(notification.description).to eq('test description')
+  end
```

Comment:
Unnecessary

Reply:
I have done it
https://github.com/labs-ruby/homeworks-2022/pull/122/commits/97b6864e45667e8fc5b0315942b4c4fe7d926808